ansible/Vagrantfile

```diff
@@ -11,6 +11,22 @@ Vagrant.configure(2) do |config|
     vb.customize ["modifyvm", :id, "--memory", "2048"]
   end
 
+  config.vm.provider :openstack do |os|
+    config.ssh.username = "centos"
```

Comment:
I think this broke non-openstack uses. Adding `node.ssh.username = "vagrant"` in the large loop corrects the issue. cc: @jburel